qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -241,19 +199,14 @@
       }
     ],
     "guppyConfig": {
-      "dataType": "subject",
-      "nodeCountTitle": "Subjects",
-      "fieldMapping": [
-        { "field": "study_submitter_id", "name": "Study" }
-      ],
+      "dataType": "location",
```

Comment:
i know this mistake is still in prod, but `location` isn't in the dictionary anymore, we might want to fix this file
maybe work with Di tomorrow?

Reply:
👍